components/layout_2020/table/mod.rs

```diff
@@ -196,7 +196,8 @@ impl TableSlotCell {
         }
     }
 
-    /// Get the node id of this cell's [`BaseFragmentInfo`]. This is used for unit tests.
```

Comment:
In Rust doc "[&#96;BaseFragmentInfo&#96;]" is a link, and unfortunately, if you remove that, there won't be any link, which is not what we want

Reply:
> In Rust doc "[`BaseFragmentInfo`]" is a link, and unfortunately, if you remove that, there won't be any link, which is not what we want

hey, just to confirm, should I keep the private link? Otherwise, I already replaced [`BaseFragmentInfo`] with its path: components/layout_2020/fragment_tree/base_fragment.rs

Reply:
I think you should be able to replace this with something like `crate::fragment_tree::BaseFragmentInfo`.

Reply:
> I think you should be able to replace this with something like `crate::fragment_tree::BaseFragmentInfo`.

Any link seems to be treated as private link and cause warning. Can I allow private links instead and keep the initial link? 

#![allow(rustdoc::private_intra_doc_links)]

Reply:
@azharcodeit Yep, you are right! The right thing to do here is to use `#![allow(rustdoc::private_intra_doc_links)]`.